hack/publishing-rules.yaml

```diff
@@ -0,0 +1,98 @@
+skip-godeps: true
+rules:
+- destination: kubernetes-code-generator
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/code-generator
+- destination: kubernetes-apimachinery
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/apimachinery
+- destination: kubernetes-api
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/api
+- destination: kubernetes-client-go
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/client-go
+- destination: kubernetes-metrics
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/metrics
+- destination: kubernetes-apiserver
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/apiserver
+- destination: kube-aggregator
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/kube-aggregator
+- destination: kubernetes-apiextensions-apiserver
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/apiextensions-apiserver
+- destination: kubernetes
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/kubernetes
+- destination: kubernetes-gengo
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: vendor/k8s.io/gengo
+- destination: google-certificate-transparency
+  branches:
+  - name: release-1.9.1
```

Comment:
release looks off

Reply:
this is fork branch name

Reply:
i want to unify the branch names so we don't use 'openshift-foo' and 'release-foo' etc. makes sense?

Reply:
> mfojtik 7 minutes ago  Member
> i want to unify the branch names so we don't use 'openshift-foo' and 'release-foo' etc. makes sense?

It doesn't make sense to track it with a kube version.  If we expect everything in the openshfit org to want the same set of patches, it should follow the original repos version.  If we expect every openshift release to want different patches, we track origin (not kube) versions.  If the upstream lacks any sense of version, then matching an openshift level where we forked it can make sense.

To my knowledge, we have nothing in that first bucket.